### PR TITLE
Use composite bundle by default

### PR DESCRIPTION
**Pull request by gurnben**
Use composite bundle by default.

### CODE REVIEW

File: snapshot.ver [hunk 1]
```diff
@@ -1 +1 @@
-1.0.0-SNAPSHOT-2020-05-12-23-58-57
+1.0.0-SNAPSHOT-2020-05-12-23-58-57
```
Comment (Kyl-Bempah) [inline snapshot.ver:1]: Do we want to update this to a 2.0 snapshot?

File: start.sh [hunk 2]
```diff
@@ -158,17 +158,24 @@ fi
...
 CUSTOM_REGISTRY_REPO=${CUSTOM_REGISTRY_REPO:-"quay.io/open-cluster-management"}
+# Default COMPOSITE_BUNDLE to true
+COMPOSITE_BUNDLE=${COMPOSITE_BUNDLE:-"true"}
 
 # If the user sets the COMPOSITE_BUNDLE flag to "true", then set to the `acm` variants of variables, otherwise the multicluster-hub version.  
 if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then OPERATOR_DIRECTORY="acm-operator"; else OPERATOR_DIRECTORY="multicluster-hub-operator"; fi;
 if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then CUSTOM_REGISTRY_IMAGE="acm-custom-registry"; else CUSTOM_REGISTRY_IMAGE="multicluster-hub-custom-registry"; fi;
 
+# Set the subscription channel, defaulted to snapshot-2.0
+if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then SUBSCRIPTION_CHANNEL="release-2.0"; else CUSTOM_REGISTRY_IMAGE="snapshot-2.0"; fi;
```
Comment (Kyl-Bempah) [inline start.sh:169]: Going to assume the naming convention is correct here.

Comment (joeg-pro) [inline start.sh:169]: else branch should be setting SUBSCRIPTION_CHANNEL, not CUSTOM_REGISTRY_IMAGE, no?

**Comment by gurnben** [comment]
See hotfix commit, fixed joe's concerns there

**Pull request merged**